docs/_docs/bestpractices/compute/compute.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: default
+parent: Best practices
+permalink: /bestpractices/compute
+nav_order: 2
+title: Compute
+author: arclares
+ms.date: 08/16/2024
+ms.service: finops
+description: 'Discover essential FinOps best practices to optimize cost efficiency and governance for your Azure resources.'
+
+---
+
+<span class="fs-9 d-block mb-4">Compute</span>
+Discover essential FinOps best practices to optimize cost efficiency and governance for your Azure compute resources.
+{: .fs-6 .fw-300 }
+
+[Share feedback](#️-looking-for-more){: .btn .fs-5 .mb-4 .mb-md-0 .mr-4 }
+
+<details open markdown="1">
+   <summary class="fs-2 text-uppercase">On this page</summary>
+
+- [🖥️ Virtual Machines](#virtual-machines)
+- [🖱️ Azure Kubernetes Service](#azure-kubernetes-service)

```

Comment:
nit: Should we sort the sections alphabetically for findability? We can do that later. Just thinking out loud.

Reply:
Good suggestion. I reorganized it.